File: src/test/java/localhost/ApplicationConfigTest.java
package localhost;

import no.nav.syfo.LocalApplication;
import no.nav.syfo.config.ApplicationConfig;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.annotation.DirtiesContext;
import org.springframework.test.context.junit4.SpringRunner;


@RunWith(SpringRunner.class)
@SpringBootTest(classes = LocalApplication.class)
@DirtiesContext
public class ApplicationConfigTest extends ApplicationConfig {

    @Test
    public void test(){
    }

}

File: src/test/kotlin/no/nav/syfo/config/ApplicationConfigTest.kt
package no.nav.syfo.config

import org.springframework.boot.test.context.SpringBootTest
import no.nav.syfo.LocalApplication
import org.junit.jupiter.api.Test
import org.junit.jupiter.api.extension.ExtendWith
import org.springframework.test.annotation.DirtiesContext
import org.springframework.test.context.junit.jupiter.SpringExtension

@ExtendWith(SpringExtension::class)
@SpringBootTest(classes = [LocalApplication::class])
@DirtiesContext
class ApplicationConfigTest {
    @Test
    fun test() {
    }
}
